extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -106,7 +106,9 @@ class AmpStickyAd extends AMP.BaseElement {
     if (scrollTop > viewportHeight) {
       this.removeOnScrollListener_();
       this.deferMutate(() => {
-        toggle(this.element, true);
+        setStyles(this.element, {
+          'visibility': 'visible',
+        });
         this.viewport_.addToFixedLayer(this.element);
         this.scheduleLayout(this.ad_);
```

Comment:
by setting `display: flex` initially, is it right that `amp-ad` will render no matter what the `amp-sticky-ad` `visibility` value is? I tried delete the `scheduleLayout` code here, and can still get ad display. Is this a desired behavior?